settings.gradle.kts

```diff
@@ -1 +1 @@
-rootProject.name = "bookstore-api"
+rootProject.name = "Bookstore-API"
```

Comment:
I tried this, and it's not a fix for the whole team, you will need to revert this, I will try to find a way to fix this in some other way

Reply:
> I tried this, and it's not a fix for the whole team, you will need to revert this, I will try to find a way to fix this in some other way

I found a way to fix this issue, but I need to confirm that this will work, please replace what you have in the `settings.gradle.kts` file with this:
```
import java.util.*

val osName = System.getProperty("os.name").lowercase(Locale.getDefault())
rootProject.name = if (osName.contains("win")) {
    "Bookstore-API"
} else {
    "bookstore-api"
}
```
and please check if this change will work. manually test it, rebuild the dependencies, and see if you can run it on your machine.
MacOS changes do work, but you never know if this does work or not.

Reply:
This change is currently working for me. There have been no issues thus far.

Reply:
> This change is currently working for me. There have been no issues thus far.

Lord can't help us with this issue rn, even some macs (Danny's mac) clones the repo with case, which makes this change useless...

Anyone any idea @airiz04 @mibal001 @dannysusername